SAKPaaS/src/assets/i18n/en.json

```diff
@@ -95,7 +95,9 @@
     "login": "Login",
     "logout": "Logout",
     "register": "Create account",
-    "guest": "Guest"
+    "guest": "Guest",
+    "level": "Level:",
+    "next-level": "experience until next level"
```

Comment:
Hier dann vielleicht auch `report(s) to level up` anstatt `experience`, das ist vielleicht ein Bisschen klarer

Reply:
Danke, hab ich vergessen zu ändern